tests/services/phase.service.spec.ts

```diff
@@ -0,0 +1,119 @@
+import { of } from 'rxjs';
+import { SessionData } from '../../src/app/core/models/session.model';
+import { UserRole } from '../../src/app/core/models/user.model';
+import { Phase, PhaseService } from '../../src/app/core/services/phase.service';
+import { UserService } from '../../src/app/core/services/user.service';
+
+const testStudent = {
+  loginId: 'testStudent',
+  role: UserRole.Student
+};
+
+const testTutor = {
+  loginId: 'testTutor',
+  role: UserRole.Tutor
+};
```

Comment:
Maybe we could define the type of object this is? Like

```typescript
const testStudent: User = {
...
};
```

or 


```typescript
const testStudent = {
...
} as User;
```

Reply:
Might be helpful for future devs who wish to add into the testcases 👍

Reply:
👍 Alright! I'll add the type declaration in it!